Sources/protoc-gen-swift/main.swift

```diff
@@ -106,7 +106,7 @@ struct GeneratorPlugin {
         + "\n"
         + "   dependencies: [\n"
         + "     .Package(url: \"https://github.com/apple/swift-protobuf\",\n"
-        + "              Version(\(SwiftProtobuf.Version.versionString))\n"
+        + "              Version(\(SwiftProtobuf.Version.versionString)))\n"
```

Comment:
I suspect we should just add a local variable nearby to hold the proper format, e.g.,
```
   let version = SwiftProtobuf.Version
   let packageVersion = "\(version.major),\(version.minor),\(version.revision)"
   ... stuff ...

   + "                     Version(\(packageVersion)))\n"
```